django_autoconfig/app_settings.py

```diff
@@ -5,6 +5,9 @@
 #: Extra URLs for autourlconf
 AUTOCONFIG_EXTRA_URLS = getattr(settings, 'AUTOCONFIG_EXTRA_URLS', ())
 
+#: A list/tuple of apps to exclude from the autourlconf
+AUTOCONFIG_EXCLUDE_URLS = getattr(settings, 'AUTOCONFIG_EXCLUDE_URLS', ())
```

Comment:
Looking further, I think my conclusion is the other setting is badly named... Let's try and make this one better :p